multichain-on-ubuntu/azuredeploy.json

```diff
@@ -0,0 +1,213 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "storageAccountNamePrefix": {
+      "type": "string",
+      "maxLength": 11,
+      "metadata": {
+        "description": "Name prefix of the Storage Account"
+      }
+    },
+    "vmDnsName": {
```

Comment:
Ok, is this a roadblock to being merged/released right now?

Is it okay we make the changes in future pull request? Thanks.



On Wednesday, January 20, 2016, Kay Singh <notifications@github.com> wrote:

> In multichain-on-ubuntu/azuredeploy.json
> <https://github.com/Azure/azure-quickstart-templates/pull/1120#discussion_r50317702>
> :
>
> > @@ -0,0 +1,213 @@
> > +{
> > +  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
> > +  "contentVersion": "1.0.0.0",
> > +  "parameters": {
> > +    "storageAccountNamePrefix": {
> > +      "type": "string",
> > +      "maxLength": 11,
> > +      "metadata": {
> > +        "description": "Name prefix of the Storage Account"
> > +      }
> > +    },
> > +    "vmDnsName": {
>
> Consider renaming to a prefix. See #966
> <https://github.com/Azure/azure-quickstart-templates/issues/966>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Azure/azure-quickstart-templates/pull/1120/files#r50317702>
> .
>